tests/Composer/Test/Satis/PackagesBuilderDumpTest.php

```diff
@@ -23,6 +23,26 @@
  */
 class PackagesBuilderDumpTest extends \PHPUnit_Framework_TestCase
 {
+    protected $package;
+
+    protected $root;
+
+    protected function setUp()
+    {
+        $this->package = new Package('vendor/name', '1.0.0.0', '1.0');
+
+        $this->root = $this->setFileSystem();
+    }
+
+    protected function setFileSystem()
```

Comment:
It's better to make this method return a `fileSystem()`. Using side-effects (e.g. changing state instead of returning something) is generally a bad idea.